includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -677,7 +677,7 @@ public function save_payment_method_checkbox( $force_checked = false ) {
 	 */
 	public function should_use_stripe_platform_on_checkout_page() {
```

Comment:
How about changing this function's name to 
```suggestion
	public function should_use_woopay_on_checkout_page() {
```
and eventually the comment describing this method?

I'm not sure where `stripe` comes from here in the name.